PProPanel/ext.js

```diff
@@ -222,11 +222,11 @@ function loadJSX() {
 	var extensionPath = csInterface.getSystemPath(SystemPath.EXTENSION);
 
 	// load general JSX script independent of appName
-	const extensionRootGeneral = extensionPath + '/jsx/';
+	var extensionRootGeneral = extensionPath + '/jsx/';
```

Comment:
Because const is an [ES6 standard](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/const), and I don't think it's supported by the CEF version used in CEP. 

But I could be mistaken, haven't tried it yet. In my projects I use babel to transpile my source code, that's why I used const there in the first place. 
This sample doesn't use any build setup, so I thought I'd better keep it clean.